jitbuilder/release/src/StructArray.cpp

```diff
@@ -34,6 +34,12 @@ void printStructElems(uint8_t type, int32_t value)
    printf("StructType { type = %#x, value = %d }\n", type, value);
    }
 
+void printElemOffsets(size_t typeOffset, size_t valueOffset)
```

Comment:
size_t is the right data type here, but I doubt Int32 always maps to "size_t" faithfully. Until you have the C type to IlType mapper in place (coming soon, I hope?), maybe the called function should take Int32's rather than size_t (I know, I know, sometimes I want size_t and sometimes I don't :) ).